ntrans_combine.py

```diff
@@ -0,0 +1,116 @@
+"""
+Reads the .csv files containing N-grams that were generated in ntrans_dataprep.py
+
+Inserts every N-gram into a collections.Counter, to determine the 10K most frequent N-grams of each sorted
+(2-grams, 3-grams, etc.).
+
+These N-grams are thereafter written to separate .csv files.
+"""
+
+import csv
+import collections
+import pathlib
+from typing import Dict, List, Tuple
+
+
+NGramCounter = Tuple[str, int]
+
+
+def delete_chunkfiles() -> None:
+    """
+    Deletes the generated chunked csv files after the finalized csv files have been finalized.
+    """
+    prompt = "Are you sure you want to delete the chunked data files? This can not be undone. (y/n) "
+    confirm = (
+        str(
+            input(
+                prompt
+            )
+        )
+        .lower()
+        .strip()
+    )
```

Comment:
Does black really do nothing about this? What if you manually put it all on the same line, and delete the first `(` and last `)`?